### PR TITLE
Profile Stats With Bio Info

### PR DESCRIPTION
**Pull request by bryananthonylewis**
This building block addition shows a profile image with stats underneath. You can also show and hide bio information under the stats.

### CODE REVIEW

**Comment by rafibomb** [comment]
@bryananthonylewis Somehow doesn't look like your commits made it into this branch. Can you push the latest commits to the branch so we can review it? Thanks!

**Comment by bryananthonylewis** [comment]
Sorry about that! Looks like the files are uploaded now.

**Comment by rafibomb** [comment]
Cool! I'm testing for merge now!

File: src/building-blocks/profile-stats/layout.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<div class="row align-center">
+  <div class="medium-8 columns">
+    <div class="card-profile">
+      <div class="card-profile__intro">
```
Comment (rafibomb) [inline src/building-blocks/profile-stats/layout.html]: use dashes for classes (consistency)

File: src/building-blocks/profile-stats/profile-stats.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: "Profile Stats"
```
Comment (rafibomb) [inline src/building-blocks/profile-stats/profile-stats.yml]: let's rename the files and the Building Block - it's a card so how about `card-profile-stats`

And also the classes should match to make it conflict with other cards doesn't the road

example: `.card-profile-stats-intro-pic`

File: src/building-blocks/profile-stats/profile-stats.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
+.card-profile {
+  border: 1px solid #cacaca;
```
Comment (rafibomb) [inline src/building-blocks/profile-stats/profile-stats.scss]: use a foundation color variable `$medium-gray` I think?

**Comment by rafibomb** [comment]
@bryananthonylewis Great Building Block! I made some comments that need to be edited before merge - ping me when they're up and merge it in!

File: src/building-blocks/profile-stats/layout.html (hunk 1 above)
Comment (bryananthonylewis) [inline src/building-blocks/profile-stats/layout.html]: Sorry was trying to follow BEM... Maybe I did it wrong? I'll change this.

**Comment by IamManchanda** [comment]
Just for references, this closes #159 on merge!

**Comment by bryananthonylewis** [comment]
Thanks @rafibomb ! I made the updates that were requested. Let me know if those look okay to you.

**Comment by IamManchanda** [comment]
@bryananthonylewis Just a lesson for your future git and github betterment, 
Please use semantic names for commit/issues/PRs specially title. The title should describe the changes you made and not what you personally done.

✅   Good:
- `Changed name to Card Profile Stats.`
- `Fix #xxx - Changed SCSS Variables to match Foundation colors.`

❌   Bad:
- **`Apply changes asked by @someone`**
- `I fixed #xxx`
- `Here some changes, btw beautiful weather today, isn't it ?`

**Comment by bryananthonylewis** [comment]
Sorry about that @IamManchanda. I'm learning. will try to keep this in mind for next time.

**Comment by IamManchanda** [comment]
Man oh Man, No issue, believe me I was in the same situation some time back... and you learn from your mistakes :D

**Comment by rafibomb** [comment]
@bryananthonylewis and @IamManchanda  You both rock! And BTW - I put my Pull Requests up for scrutiny all the time. We're all learning together to be better and more efficient. Great work!

**Pull request merged**